homework_4/task_4.py

```diff
@@ -0,0 +1,14 @@
+while True:
+    n_term = int(input("Enter n between [0, 20]: "))
+    if 0 <= n_term <= 20:
+        break
+
+first = 0 
+second = 1
+
+for i in range(2, n_term + 1):
+    next_ = first + second
+    first = second
+    second = next_
+
+print(next_) 
```

Comment:
სწორად იმუშავებს, უბრალოდ არაა მიღებული  პრაქტიკა რომ for-ის scope-ში აღწერილი ცვლადი for-ის გარეთ გამოიყენო
https://www.programiz.com/python-programming/global-local-nonlocal-variables

ამ შემთხვევაში next_ იქმნება for-ის ტანში და ბეჭდავ for-ის გარეთ

ჯობია first, second-თან ერთად აგეღწერა next_ და ბოლო მნიშვნელობაც მაგ ცვლადში დამახსოვრდებოდა და მერე ის დაგებეჭდა